spec/support/email_helper.rb

```diff
@@ -0,0 +1,11 @@
+module OpenFoodNetwork
+  module EmailHelper
+    # Some specs trigger actions that send emails, for example creating an order.
+    # But sending emails doesn't work out-of-the-box. This code sets it up.
+    # It's here in a single place to allow an easy upgrade to Spree 2 which
```

Comment:
I wouldn't mention "spree upgrade" in comments any more. It will be past and outdated soon :-D

Reply:
I'd remove the 3rd and 4th line.

Reply:
We need to change this method in the Spree branch once this is all merged. I would remove those lines then. Having it in the master branch will make everybody aware that they shouldn't temper with this code during the Spree upgrade unless there is a really good reason -> avoiding conflicts.